File: comparing sets.cs


HashSet<int> set = new HashSet<int>();
HashSet<int> set2 = new HashSet<int>();

set.Add(1);
set.Add(3);
set.Add(4);

foreach (int item in (new int[] { 1, 3, 4}))
{
    set2.Add(item);
}

Console.WriteLine("number of elements in set=" + set.Count);
Console.WriteLine(String.Format("Are the sets equal? {0} ", set.SetEquals(set2)));

File: comparing sets.rb
require 'set'

set = Set.new
set2 = Set.new

set.add 1
set.add 3
set.add 4

[1, 3, 4].each{|item|set2.add item}




puts "Number of elements in the set is #{set.count}"
puts "Are the sets equal? #{set==set2}"
